Tests/UInt128Tests/UInt128Tests.swift

```diff
@@ -497,7 +497,7 @@ class BinaryIntegerTests : XCTestCase {
         let testResult = UInt128(upperBits: upperBits, lowerBits: lowerBits)
 
         for index in 0 ... UInt128.bitWidth / UInt.bitWidth {
-            let currentWord = testResult._word(at: index)
+            let currentWord = testResult.words[index]
```

Comment:
Since we already have an array here, and each word in the array should still equal the magic number of `4294967297`, would you mind changing the `for` loop to a `testResult.words.forEach` functional call?